### PR TITLE
Transpile new DataView operations

### PR DESCRIPTION
**Pull request by mathiasbynens**
Issue: #12
Issue: https://github.com/GoogleChromeLabs/jsbi/pull/87

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -86,6 +95,19 @@ export default function(babel) {
               t.identifier('BigInt'), t.identifier(name)),
           args);
     }
+    if (dataViewMethods.has(name)) {
+      if (args.length !== 2 && args.length !== 3) {
+        throw path.buildCodeFrameError(
+            `${name} must have exactly two or three arguments`);
```
Comment (jakobkummerow) [inline src/index.js:101]: This isn't quite correct: the two `...Get...` methods take 2 or 3 arguments, the two `...Set...` methods take 3 or 4. In all cases, `littleEndian` is optional.
I think in the interest of simplicity it's probably fine to just extend this check to accept 2-4 arguments for all methods.

File: test/fixtures/full-import-specifier/expected.js [hunk 2]
```diff
@@ -49,3 +49,11 @@ Number(a);
 typeof a === "bigint";
 BigInt.asIntN(64, 42n);
 BigInt.asUintN(64, 42n);
+const buffer = new ArrayBuffer(16); // 0x7FFFFFFFFFFFFFFFn, the highest possible BigInt value that fits in
```
Comment (jakobkummerow) [inline test/fixtures/full-import-specifier/expected.js:52]: It's weird that the line break before the comment is removed, but that seems unrelated to this patch.

**Review by jakobkummerow — APPROVED** [review]
Thanks a lot! LGTM with a comment.

File: src/index.js [hunk 3]
```diff
@@ -51,6 +51,15 @@ const staticMethods = new Set([
   'asUintN',
 ]);
 
+// TODO: docs
+// https://github.com/GoogleChromeLabs/jsbi/pull/87
```
Comment (mathiasbynens) [inline src/index.js:55]:
```suggestion
// https://github.com/GoogleChromeLabs/jsbi#how
```

File: src/index.js (hunk 1 above)
Comment (mathiasbynens) [reply src/index.js:101]:
```suggestion
      if (2 <= args.length && args.length <= 4) {
        throw path.buildCodeFrameError(`${name}: incorrect number of arguments`);
```

File: test/fixtures/full-import-specifier/expected.js (hunk 2 above)
Comment (mathiasbynens) [reply test/fixtures/full-import-specifier/expected.js:52]: Yeah, Babel’s pretty-printing does not try to match the source format. It doesn’t matter too much though since this probably gets minified in practice afterwards anyways.
